src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,7 +48,10 @@ public RobotContainer() {
         .oddSquare(Utils.deadZone(-m_driveController.getRightX(), OIConstants.kJoystickDeadzone))
         * SwerveConstants.kMaxAngularSpeedRadiansPerSecond;
     m_swerveDriveSubsystem.setDefaultCommand(
-        new SwerveDriveCommand(m_swerveDriveSubsystem, x, y, rot, () -> m_driveController.getRightBumper()));
+        new MoveCommand(m_swerveDriveSubsystem)
+            .withXSpeedSupplier(x)
+            .withYSpeedSupplier(y)
+            .withRotSpeedSupplier(rot));
```

Comment:
we could add a boolean supplier to return whether the left bumper was pressed for field relative mode

Reply:
we'd just set it to be default true in movecommand but it could be changed by calling a method, like with x, y and rotation

Reply:
This is a good idea but we would need to add some math for when the robot is set to drive in robot relative mode